build/transpile.js

```diff
@@ -644,7 +647,7 @@ class Transpiler {
             // a proper \ccxt\Exchange::deep_extend() base method is implemented instead
             // [ /this\.deepExtend\s/g, 'array_replace_recursive'],
             [ /(\w+)\.shift\s*\(\)/g, 'array_shift($1)' ],
-            [ /(\w+)\.reverse\s*\(\)/g, 'array_reverse($1)' ],
```

Comment:
we can keep it no worries, in some special case we might need it

Reply:
ok, i've added python version and included comments.